vss-processor/src/main/kotlin/org/eclipse/kuksa/vssprocessor/spec/VssNodeSpecModel.kt

```diff
@@ -166,40 +171,63 @@ internal class VssNodeSpecModel(
             .build()
     }
 
-    private fun createVssSignalSpec(): Triple<ParameterizedTypeName, MutableList<PropertySpec>, ParameterSpec?> {
-        val propertySpecs = mutableListOf<PropertySpec>()
-        var parameterSpec: ParameterSpec? = null
+    private fun createVssSignalSpec(): Triple<ParameterizedTypeName, List<PropertySpec>, List<ParameterSpec>> {
```

Comment:
still a lot can be improved here, but that will be a bigger refactoring which I don't dare to do in this ticket